# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libstdc++-v3/fixincl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No fixinclude neded and wanted.

  - Rene Rebe <rene@exactcode.de>

--- ../gcc/fixinc/fixincl.sh.orig
+++ ../gcc/fixinc/fixincl.sh
@@ -143,6 +143,8 @@
     INPUTLIST="$@"
 fi
 
+INPUTLIST=""
+
 for INPUT in ${INPUTLIST} ; do
 
 cd ${ORIGDIR}
